src/com/capstone/screens/BookCarScreen.java

```diff
@@ -8,7 +8,12 @@
 import java.util.UUID;
 
 public class BookCarScreen implements IScreen {
-    private final BookingService bookingService = new BookingService(new BookingArrayDataAccess());
```

Comment:
Have to ensure that this class will only be instantiated once, or else booking[0] and booking[1] will be overwritten if there were changes

Read into singleton pattern